### ISSUE TITLE
[$30][MSFT-112] Member registration page: The help content related to the 'email' 'user name' and 'password' fields is not announced when user navigates with tab key.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2813]
**Steps to Reproduce**
1. Go to the Members registration page:https://accounts.topcoder.com/member/registration. 2. Ensure that the screen reader is switched on. 3. Press tab till your arrive at the 'enter your email id' text box. 4. Notice that the screen reader doesn't announce the help content. 5. Press tab to arrive at the 'user name' and 'password' text fields and observe that the help content is skipped.

**Expected Result**
The help content should be announced by screen readers when user navigates through the form fields with tab key. Since the help content is not tagged to the respective fields, the announcement is not made.

**Actual Result**
Screen reader doesn't announce the help content associated with the 'email' 'user name' and 'password' fields when user navigates with tab key alone. There is no indication to the user as to the presence of such help content unless and until he/she navigates with arrow keys.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2017

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096271 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096271 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096271 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096271 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097185 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097185 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Oanh-and-only-Oanh** [comment]
please review code PR appirio-tech/accounts-app#256

**Comment by Oanh-and-only-Oanh** [comment]
QA fail- announces the field names but NOT the help text for the fields

**Comment by afrisalyp** [comment]
@Oanh-and-only-Oanh Can you please give us more detail how you reproduce the failure, what's the browser and the screen reader app?

The reviewer has accepted the solution.
https://github.com/appirio-tech/accounts-app/pull/256

**Comment by drasticdpk** [comment]
@afrisalyp @Oanh-and-only-Oanh @mishacucicea  its very hard to me to read out the help content of user name and password. Can we make this pop up read out like  date range filed in challenge page ?
Here is the reference video :
https://monosnap.com/file/iWLaPA6p52IbcXYHvAEIMV7HmrwOag
Test platform : Windows/Chrome/JAWS

**Comment by nkumar-topcoder** [comment]
each reviewer have different outcome

**Comment by afrisalyp** [comment]
@drasticdpk @nkumar-topcoder 
The test is against topcoder production, right?

But I don't see my PR has been merged.
https://github.com/appirio-tech/accounts-app/pull/256
It still open.

**Comment by mishacucicea** [comment]
@drasticdpk It would seem fine for me not sure exactly what's expected, it would seems the fix is satisfying the requirements. Note that when I listen to your video it seems like it has a double-voice on recording. Also, how to you enable the keyboard/calendar tips?

**Comment by drasticdpk** [comment]
@mishacucicea ,
1. When user reached to userName/Email/Password control, screen reader does not read that popup automatically. In video, I randomly (which is not correct way) pressed the H key and able to read that popup.
2. Due to recoding from monosnap we are listening double-voice some control, in real it is not.
3. I am using screen reader JAWS,  while reaching cursor  date time picker, if we press '?' from keyboard it will show/speak keyboard/calendar tips.
![image](https://user-images.githubusercontent.com/25297859/62407103-bcfd7e80-b5d2-11e9-8088-fe1d64fdde8e.png)

@afrisalyp , we are having this issue on both Test/Prod environment.

**Comment by lakshmiathreya** [comment]
Verified MacOS/Chrome/Voice Over and Win10/Chrome/NVDA - help content related to email, username and password read out correctly.

**Comment by tosha5252** [comment]
Verified using Win10/Chrome/NVDA

@nkumar-topcoder

**Comment by tosha5252** [comment]
Verified using Win10/Chrome/NVDA/JAWS

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097185<br/><br/>```This is an automated message for crazyk via Topcoder X```
